coalib/results/Diff.py

```diff
@@ -244,10 +243,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._generate_linebreaks(self._file)
```

Comment:
definitely cache this! This is quite cost intensive if a user needs it several times. Either use `lru_cache(1)` or save it directly on instantiation inside a variable :+1: